CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ All notable changes to this project will be documented in this file.
 -   #1472 : Add C printing support for a class containing scalar data.
 -   #1492 : Types of arguments for class methods can be declared like function arguments.
 -   #1511 : Add support for the `cmath` library.
--   #1508 : Add C support for a class destructor.
+-   #1508 : Add C support for a class `Destructor`.
```

Comment:
This word is valid. It is not code. Please change this back and add it to the file `.dict_custom.txt`

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1508/commits/f0373f446e6206f5fcbc5676ce6f0d28d9cbff05.